src/bindings/python/tests/test_inference_engine/test_infer_request.py

```diff
@@ -298,16 +300,29 @@ def test_query_state_write_buffer(device, input_shape, data_type, mode):
             mem_state.state = tensor
 
             res = request.infer({0: np.full(input_shape, 1, dtype=data_type)})
+            k = list(res.keys())[0]
             expected_res = np.full(input_shape, 1 + const_init, dtype=data_type)
         elif mode == "reset_memory_state":
             # reset initial state of ReadValue to zero
             mem_state.reset()
             res = request.infer({0: np.full(input_shape, 1, dtype=data_type)})
-
+            k = list(res.keys())[0]
             # always ones
             expected_res = np.full(input_shape, 1, dtype=data_type)
         else:
             res = request.infer({0: np.full(input_shape, 1, dtype=data_type)})
+            k = list(res.keys())[0]
```

Comment:
Why this line is copied over 3 times in different branches?

Reply:
Fixed.